qc_grader/challenges/fall_2022/lab3.py

```diff
@@ -30,7 +30,8 @@ def grade_lab3_ex1(attempt_graph: Graph, attempt_n: int) -> None:
 
 @typechecked
 def grade_lab3_ex2(attempt_ising: PauliSumOp) -> None:
-    grade(attempt_ising, 'ex3-2', _challenge_id)
+    answer = paulisumop_to_json(attempt_ising)
```

Comment:
when passing a single object the client already knows how to serialize, we dont need to call the `*_to_json` function